### PR TITLE
clean warnings

### PR DESCRIPTION
**Pull request by mosheraboh**

### CODE REVIEW

File: mammal/examples/tests/test_dti_bindingdb_kd.py [hunk 1]
```diff
@@ -61,7 +61,7 @@ def test_evaluate(model_dir: str):
         "evaluate=True",
         f"model.pretrained_kwargs.pretrained_model_name_or_path={model_dir}/best_epoch.ckpt",
     ]
-    with hydra.initialize_config_dir(TEST_CONFIG_DIRPATH):
+    with hydra.initialize_config_dir(TEST_CONFIG_DIRPATH, version_base="1.1"):
```
Comment (SagiPolaczek) [inline mammal/examples/tests/test_dti_bindingdb_kd.py:64]: Do the other hydra initializations mention the version? Surprising that this is the only change :)

File: mammal/model.py [hunk 2]
```diff
@@ -408,7 +408,7 @@ def from_pretrained(
                 config.override(config_overrides)
             model = cls(config)
 
-            pl_ckpt_dict = torch.load(pretrained_model_name_or_path, map_location="cpu")
+            pl_ckpt_dict = torch.load(pretrained_model_name_or_path, map_location="cpu", weights_only=True)
```
Comment (SagiPolaczek) [inline mammal/model.py:411]: Is the one working for you flawlessly? I think I had issues with it 🤔

**Review by SagiPolaczek — APPROVED** [review]
The changes looks good!

I left two comments inline :-)

File: mammal/examples/tests/test_dti_bindingdb_kd.py (hunk 1 above)
Comment (mosheraboh) [reply mammal/examples/tests/test_dti_bindingdb_kd.py:64]: Surprisingly yes

File: mammal/model.py (hunk 2 above)
Comment (mosheraboh) [reply mammal/model.py:411]: At least the tests are working.
Do you remember what was your issue? plus remmber that we are not storing config inside the checkpoint anymore.

**Review by matanninio — COMMENTED** [review]
nothing to add

**Pull request merged**